src/components/auth/RegisterFoundation.tsx

```diff
@@ -0,0 +1,158 @@
+"use client";
+import Button from "@/components/ui/Button";
+import Input from "@/components/ui/Input";
+import Image from "next/image";
+import Link from "next/link";
+import { useState } from "react";
+import Select from "../ui/Select";
+import TextArea from "../ui/TextArea";
+
+export default function RegisterFoundation() {
+    const [name, setName] = useState("");
+    const [description, setDescription] = useState("");
+    const [category, setCategory] = useState("");
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
```

Comment:
```suggestion
    const [name, setName] = useState("");
    const [description, setDescription] = useState("");
    const [category, setCategory] = useState("");
    const [email, setEmail] = useState("");
    const [password, setPassword] = useState("");
```

Please, let's remove the logic for form submission. This will be another issue for someone else in the future. For now we just want the design.